spec/services/iiif_print/manifest_builder_service_behavior_decorator_spec.rb

```diff
@@ -0,0 +1,169 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe IiifPrint::IiifManifestPresenterBehavior do
+  let(:presenter) { double(Hyrax::IiifManifestPresenter) }
+
+  describe '#sanitize_v2' do
+    context 'when thumbnail files are present' do
+      let(:service) { Hyrax::ManifestBuilderService.new }
+      # TODO: REFACTOR!
+      let(:manifest_w_thumbnail_hash) do
```

Comment:
This spec is sloppy but captures the desired behavior for now. Given the time crunch we're in, we've decided to proceed with a TODO to revisit later. It is passing as written; it's just not ideal.